### PR TITLE
[vscode] Workaround broken activation for `.mv` files

### PR DESCRIPTION
**Pull request by ejgallego**
Suggested by Tomer Chachamu (thanks!) in
https://github.com/microsoft/vscode-discussions/discussions/901

Should fix issue #596 , however I'm not marking this as closed, as this workaround could actually be due to a VSCode bug.

Thanks to Théo Zimmerman for the report.

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -10,6 +10,9 @@
  - fix hover position computation on the presence of Utf characters
    (@ejgallego, #597, thanks to Pierre Courtieu for the report and
    example, closes #594)
+ - workaround seemingly VSCode activation bug that prevented extension
+   activation (@ejgallego @r3m0t, #598, cc #596, reported by Théo
+   Zimmerman)
```
Comment (Zimmi48) [inline CHANGES.md:15]: You should probably edit the changelog entry if this is not a VS Code bug and the fix here is the proper fix as opposed to a workaround.

Comment (ejgallego) [reply CHANGES.md:15]: Indeed, thanks for pointing it out!

**Comment by Zimmi48** [comment]
There is a serious issue though with pretending `mv` are just Markdown files and then associating coq-lsp with Markdown files: when opening a non-`mv` file (or even a Jupyter notebook with Markdown cells), the coq-lsp extension ends up being triggered and opening its Goals window. In my course where I use Coder, I have a first practical session, in the same VS Code workspace, which is not about Coq and uses a Jupyter notebook. I noticed that the Goals window kept appearing (even after being closed) when students interacted with the notebook (more specifically when interacting with the Markdown cells). I wonder if there isn't a solution that would e.g., add a new alias like CoqMarkdown for the `mv` files and then use this alias to activate the Coq extension...

**Comment by ejgallego** [comment]
@Zimmi48 what VSCode version are you on?

**Comment by ejgallego** [comment]
I did test for this particular problem before closing this issue, but indeed it seems I didn't test for this properly; I can observe the behavior you are pointing out.

I have prepared a fix at least for the Goals window issue, see #625 

Indeed the situation is maybe still not good? It seems to me upstream is still evolving activation rules; I'd be great if someone could gather more info; I'm afraid I won't be able to interact more with them in the short term.

**Pull request merged**